src/SFA.DAS.ApplyService.Web/ViewModels/Roatp/TaskListViewModel.cs

```diff
@@ -143,5 +298,23 @@ public bool ApplicationSequencesCompleted()
 
             return true;
         }
+
+        private string ConvertTaskListSectionStatusToCssClass(string sectionStatus)
+        {
+            if (sectionStatus == TaskListSectionStatus.Blank)
```

Comment:
this should be a switch statement

Reply:
Case only works with numerics, so converted to an else if